### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by CreepyMo**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+        pytest.param(
+            27, 27, [2, 2],
+            id="27 cat/dog years should convert into 2 human age."
+        ),
+        pytest.param(
+            28, 28, [3, 2],
+            id="28 cat/dog years should convert into 3/2 human age."
+        )
+    ]
+)
```
Comment (Dimosphen1) [inline app/test_main.py:37]: Add tests that pass large numbers, negative numbers, and different numbers to `cat_age` and `dog_age` as mentioned in the checklist. Besides that, declare another function to test against the raised error in case different invalid data types are passed to the function

File: app/test_main.py [hunk 2]
```diff
@@ -1,3 +1,43 @@
+import pytest
```
Comment (Dimosphen1) [inline app/test_main.py:1]: Add a blank line between different groups of imports according to PEP-8

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

File: app/test_main.py (hunk 1 above)
Comment (CreepyMo) [reply app/test_main.py:37]: Done

File: app/test_main.py (hunk 2 above)
Comment (CreepyMo) [reply app/test_main.py:1]: Done
